.github/workflows/ci.yml

```
@@ -0,0 +1,31 @@
+# Run basic tests for this app on the latest aiidalab-docker image.
+
+name: continuous-integration
+
+on:
+  [push]
```

Is there a reason why we're not testing on PRs?
```suggestion
on: [push, pull_request]
```

---

No.